resources/default-settings/evdev.example.yaml

```diff
@@ -0,0 +1,59 @@
+devices: # A list of evdev devices each containing one or multiple input/output devices
+  joystick: # A nickname for a device
+    device_name: DragonRise Inc.   Generic   USB # Device name
```

Comment:
Is this just a pretty name or actually relevant to access/call the device?

Reply:
The 'device_name' is relevant as it is used to get the device number (in contrast to the nickname line 2 which is cosmetic).

The docs contain a section how to look it up: https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/49839adb01019cb296d67dc29c06cd9a6af0bc62/documentation/builders/event-devices.md#identifying-the-device_name